src/Jackalope/Version/VersionHandler.php

```diff
@@ -74,12 +76,14 @@ public function addVersionProperties(NodeInterface $node)
         $additionalOperations[] = new AddNodeOperation($versionLabels->getPath(), $versionLabels);
         $rootVersion = $versionHistory->addNode('jcr:rootVersion', 'nt:version');
         $rootVersion->setProperty('jcr:uuid', UUIDHelper::generateUUID());
+        $rootVersion->setProperty('jcr:predecessors', array());
+        $rootVersion->setProperty('jcr:successors', array()); // not part of the specification, but seems to be required
```

Comment:
The `VersionHandler` won't be instantiated for jackalope-jackrabbit, so this shouldn't be an issue.